src/System.Data.SqlClient/src/System/Data/SqlClient/LocalDBAPI.cs

```diff
@@ -101,6 +101,7 @@ private static LocalDBFormatMessageDelegate LocalDBFormatMessage
         {
             get
             {
+#if !MANAGED_SNI
```

Comment:
I commented out all references to api-win*.dll. This code isn't getting used in Managed SNI anyway.